### PR TITLE
Export cdk-spot-one Security Group , can add custom port.

### PR DESCRIPTION
**Pull request by guan840912**
- Export cdk-spot-one Security Group , can add custom port.
- add test stack

### CODE REVIEW

File: src/spot.ts [hunk 1]
```diff
@@ -203,6 +203,10 @@ export class SpotFleet extends Resource {
    * The time when the the fleet allocation will expire
    */
   private validUntil?: string;
+  /**
+   * @default - open SSH 22 port Any IP .
+   */
+  public readonly defaultSG: ec2.ISecurityGroup;
```
Comment (pahud) [inline src/spot.ts:209]: How about `defaultSecurityGroup` ?

File: src/spot.ts [hunk 2]
```diff
@@ -203,6 +203,10 @@ export class SpotFleet extends Resource {
    * The time when the the fleet allocation will expire
    */
   private validUntil?: string;
+  /**
+   * @default - open SSH 22 port Any IP .
```
Comment (pahud) [inline src/spot.ts:207]: How about just 

> The default security group of the instance, which only allows TCP 22 SSH ingress rule.

**Review by pahud — CHANGES_REQUESTED** [review]
Thanks. Just a few minor changes required.

File: src/spot.ts (hunk 2 above)
Comment (pahud) [reply src/spot.ts:207]: Just

```ts
/**
 * The default security group of the instance, which only allows TCP 22 SSH ingress rule.
**/
```

Comment (guan840912) [reply src/spot.ts]: no problem 🤟🏻 , I will update Description .

File: src/spot.ts (hunk 1 above)
Comment (guan840912) [reply src/spot.ts]: Okay , Easy to know what it means 👌 👌 👌

File: src/spot.ts [hunk 3]
```diff
@@ -203,6 +203,10 @@ export class SpotFleet extends Resource {
    * The time when the the fleet allocation will expire
    */
   private validUntil?: string;
+  /**
+   * @default - The default security group of the instance, which only allows TCP 22 SSH ingress rule.
```
Comment (pahud) [inline src/spot.ts:207]: Let's remove the `@default` tag. This tag is only used with the optional property.

Just the description would be enough.

**Review by pahud — CHANGES_REQUESTED** [review]

File: src/spot.ts (hunk 3 above)
Comment (guan840912) [reply src/spot.ts]: no problem !!!  my mistake ... XD

**Review by pahud — APPROVED** [review]

**Pull request merged**